### PR TITLE
elasticsearch: collect metrics from _cat/indices

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

Fixes: #6541

This PR adds per index metrics:
 - docs count
 - store size
 - number of replicas
 - health status

Modules gathers these metrics from `_cat/indices` endpoint. Per index metrics is optional and disabled by default.

##### Component Name

[`/collectors/python.d.plugin/elasticsearch`](https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/elasticsearch/elasticsearch.chart.py)

### CODE REVIEW

**Comment by ilyam8** [comment]
@Wing924 could you test the PR?

> elasticsearch_index_replica{dimension="twitter"} 5
# gauge red=0, yellow=1, green=2

I did
 - green: 0
 - yellow: 1
 - red: 2


Reason - netdata hides zeros by default, it would be wrong to have red status hidden.

**Comment by Wing924** [comment]
@ilyam8 
スクリーンショット 2019-09-30 22 03 50
This module become very slow after apply this change.
I think query every seconds is too hard for Elasticsearch.

**Comment by ilyam8** [comment]
@Wing924 `elasticsearch` module does queries every 5 sec by default.

File: collectors/python.d.plugin/elasticsearch/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # elasticsearch
 
-This module monitors Elasticsearch performance and health metrics.
+This module monitors `Elasticsearch` performance and health metrics.
```
[the remark this answers is not in the document]
Comment (ilyam8) [reply collectors/python.d.plugin/elasticsearch/README.md]: yes, good idea

File: collectors/python.d.plugin/elasticsearch/elasticsearch.chart.py [hunk 2]
```diff
@@ -550,88 +616,131 @@ def _get_data(self):
...
+        if not parsed:
             return queue.put(dict())
 
-        data = self.json_reply(raw_data)
+        data = fetch_data(raw_data=parsed, metrics=HEALTH_STATS)
+        dummy = {
+            'status_green': 0,
+            'status_red': 0,
+            'status_yellow': 0,
+            'status_foo1': 0,
```
Comment (Wing924) [inline collectors/python.d.plugin/elasticsearch/elasticsearch.chart.py]: debug code?

Comment (ilyam8) [reply collectors/python.d.plugin/elasticsearch/elasticsearch.chart.py]: kind of 😄

**Comment by ilyam8** [comment]
> docs count unit is wrong?

Looks ok, try ti disable `Scale Units` option (Settings->Locale)

**Pull request merged**